repos/fdbt-site/src/pages/api/matching.ts

```diff
@@ -45,7 +45,7 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
             return;
         }
 
-        if (carnetFareType) {
+        if (carnetFareType !== undefined && carnetFareType) {
```

Comment:
What's the reason for this?

Reply:
@RobCatton I want to say the reason is so it only does the thing if carnetFareType exists and is defined

Reply:
its because the contents of the carnetFareType (if not undefined) is a boolean, but it can be present (not undefined) and also false. So this check ensures its there, and true.

Reply:
I think that's logically equivalent to
`if (carnetFareType)`
?